sqlmesh/core/context.py

```diff
@@ -858,29 +858,26 @@ def format(
         append_newline: t.Optional[bool] = None,
         **kwargs: t.Any,
     ) -> None:
-        """Format all SQL models."""
-        for model in self._models.values():
-            if not model._path.suffix == ".sql":
+        """Format all SQL models and audits."""
+        combined_models_audits = {**self._models, **self._audits}
+        for node in combined_models_audits.values():
```

Comment:
I agree wasn't feeling `node` either too much so I could use `target` that you suggest. I was also considering simply  leaving it as `model` since audits inherits from `PydanticModel`

Reply:
`model` has the same issue with `node`, it's overloaded

Reply:
yes that's true your suggestion makes more sense in this context, renaming it to this